src/calculator.js

```diff
@@ -6,7 +6,7 @@ exports._check = () => {
   
 };
 
-exports.add = (x, y) => {
+exports.add = (x,y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule